# Variance of 2D bounding box.
#

float64 var_x1
float64 var_y1
float64 var_x2
float64 var_y2
